mne/simulation/tests/test_raw.py

```diff
@@ -483,21 +483,19 @@ def test_simulate_raw_chpi():
     assert_allclose(raw_sim[hpi_pick][0], 0.)
     assert_allclose(raw_chpi[hpi_pick][0], hpi_ons.sum())
     # test that the cHPI signals make some reasonable values
-    picks_meg = pick_types(raw.info, meg=True, eeg=False)
-    picks_eeg = pick_types(raw.info, meg=False, eeg=True)
-
-    for picks in [picks_meg[:3], picks_eeg[:3]]:
+    picks_meg = pick_types(raw.info, meg=True)[:3]
+    picks_eeg = pick_types(raw.info, eeg=True)[:3]
+    for picks in (picks_meg, picks_eeg):
         psd_sim, freqs_sim = (
             raw_sim.compute_psd(picks=picks).get_data(return_freqs=True))
         psd_chpi, freqs_chpi = (
             raw_chpi.compute_psd(picks=picks).get_data(return_freqs=True))
-
         assert_array_equal(freqs_sim, freqs_chpi)
-        freq_idx = np.sort([np.argmin(np.abs(freqs_sim - f))
-                            for f in hpi_freqs])
+        # bins closest to cHPI freqs should have very high energy in MEG chans
         if picks is picks_meg:
```

Comment:
incidentally, since #4322 this conditional was never getting hit because `picks_meg` was all MEG picks, but that PR changed `picks` to be just the first 3 entries of it.